migrate-mongo-config.js

```diff
@@ -0,0 +1,26 @@
+/* eslint-disable @typescript-eslint/no-var-requires */
+require("dotenv").config();
```

Comment:
Do we need dotenv? If we don't need it please remove it also from package.json

Reply:
the migrate-mongo-config won't work if we delete the dotenv.